modules/gdscript/register_types.cpp

```diff
@@ -143,9 +142,4 @@ void unregister_gdscript_types() {
 
 	ResourceSaver::remove_resource_format_saver(resource_saver_gd);
 	resource_saver_gd.unref();
-
-#ifdef TOOLS_ENABLED
-	EditorTranslationParser::get_singleton()->remove_parser(gdscript_translation_parser_plugin, EditorTranslationParser::STANDARD);
-	gdscript_translation_parser_plugin.unref();
```

Comment:
Removing this lines caused (again) leak of GDScriptEditorTranslationParserPlugin